src/background/cryptKeeper.ts

```diff
@@ -32,19 +33,22 @@ export default class CryptKeeperController extends Handler {
 
   private backupService: BackupService;
 
+  private keyStorageService: KeyStorageService;
+
   constructor() {
     super();
     this.requestManager = new RequestManager();
     this.zkIdentityService = ZkIdentityService.getInstance();
     this.approvalService = ApprovalService.getInstance();
-    this.walletService = WalletService.getInstance();
+    this.miscStorageService = MiscStorageService.getInstance();
     this.lockService = LockerService.getInstance();
     this.browserService = BrowserUtils.getInstance();
     this.historyService = HistoryService.getInstance();
     this.backupService = BackupService.getInstance()
       .add(BackupableServices.APPROVAL, this.approvalService)
       .add(BackupableServices.IDENTITY, this.zkIdentityService)
       .add(BackupableServices.LOCK, this.lockService);
+    this.keyStorageService = KeyStorageService.getInstance();
```

Comment:
Maybe just move this line `this.keyStorageService = KeyStorageService.getInstance();` up before `this.backupService = BackupService.getInstance()` so we have in the end the backup listeners.